src/common/components/button/button.css

```diff
@@ -1,69 +0,0 @@
-$primary-button-color: $ubuntu-orange;
```

Comment:
The existing Button component has been relocated to the vanilla directory, and replaces the new component I was using.